From 56db085fc94aed2a22bcc70f2c0441a0852c780f Mon Sep 17 00:00:00 2001
From: ViKing <viking.yfy@qq.com>
Date: Fri, 28 Aug 2026 19:39:13 +0800
Subject: [PATCH] dt-bindings: net: dsa: add Qualcomm tag protocols

List the Qualcomm four-byte and 802.1Q transports as valid alternate
DSA tag protocols.

Signed-off-by: ViKing <viking.yfy@qq.com>
---
 Documentation/devicetree/bindings/net/dsa/dsa-port.yaml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/Documentation/devicetree/bindings/net/dsa/dsa-port.yaml
+++ b/Documentation/devicetree/bindings/net/dsa/dsa-port.yaml
@@ -48,7 +48,6 @@ properties:
   dsa-tag-protocol:
     description:
       Instead of the default, the switch will use this tag protocol if
-      - qca_8021q
       possible. Useful when a device supports multiple protocols and
       the default is incompatible with the Ethernet device.
     enum:
@@ -56,6 +55,8 @@ properties:
       - edsa
       - ocelot
       - ocelot-8021q
+      - qca_4b
+      - qca_8021q
       - rtl8_4
       - rtl8_4t
       - seville
